onnx2torch/node_converters/gather.py

```diff
@@ -76,6 +78,61 @@ def symbolic(graph: torch_C.Graph, *args) -> torch_C.Value:
         return graph.op('Gather', input_tensor, indices, axis_i=axis, outputs=1)
 
 
+class OnnxGatherND(nn.Module, OnnxToTorchModuleWithCustomExport):
+    """ONNX GatherND implementation."""
+
+    def __init__(self, batch_dims: int = 0):
+        super().__init__()
+        self.batch_dims: int = batch_dims
+
+    def forward(self, input_tensor: torch.Tensor, indices: torch.Tensor) -> torch.Tensor:  # pylint: disable=C0116
+        def _forward():
+            return self._gather_nd(data=input_tensor, indices=indices, batch_dims=self.batch_dims)
+
+        if torch.onnx.is_in_onnx_export():
+            if get_onnx_version() == 11:  # Special case: opset 11, without batch_dims parameter.
+                if self.batch_dims != 0:
+                    raise ValueError(f'GatherND from opset 11 does not support batch_dims != 0, got {self.batch_dims}')
+                return _GatherNDExportToOnnx.set_forward_and_apply(_forward, input_tensor, indices)
+
+            return _GatherNDExportToOnnx.set_forward_and_apply(_forward, input_tensor, indices, self.batch_dims)
+
+        return _forward()
+
+    @staticmethod
+    def _gather_nd(data: torch.Tensor, indices: torch.Tensor, batch_dims: int) -> torch.Tensor:
+        if batch_dims != 0:
+            raise NotImplementedError('GatherND for batch_dims != 0 is not implemented')
+
+        r, m = len(data.shape), indices.shape[-1]  # pylint: disable=C0103
+        if m > r or m < 1:
+            raise ValueError(
+                f'The last dimension of indices should have a value between 1 (inclusive) and data rank (inclusive), '
+                f'got {m} and {r} respectively'
+            )
+
+        total_samples = indices.shape[:-1].numel()
+        output_shape = indices.shape[:-1] + data.shape[m:]
+        _indices = torch.split(
```

Comment:
underscore is not necessary by 2 reasons:
1) if you want to make variable with similar purpose/logic you should use underscore in the end of name
2) actually you have no reasons to use new name here